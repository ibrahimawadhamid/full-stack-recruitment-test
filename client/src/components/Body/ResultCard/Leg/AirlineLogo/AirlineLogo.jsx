import React, {Component} from 'react';
import {BpkGridColumn} from 'bpk-component-grid';
import BpkImage from 'bpk-component-image';

class AirlineLogo extends Component {
  render() {
    return (
      <BpkGridColumn width={2}>
        <BpkImage altText="airline logo" height={48} width={48} style={{width: 48}} src="./EZ.png"/>
      </BpkGridColumn>
    );
  };
}

export default AirlineLogo;
