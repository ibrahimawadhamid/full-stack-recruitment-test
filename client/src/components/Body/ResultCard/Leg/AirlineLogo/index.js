import AirlineLogo from './AirlineLogo';

export default AirlineLogo;
