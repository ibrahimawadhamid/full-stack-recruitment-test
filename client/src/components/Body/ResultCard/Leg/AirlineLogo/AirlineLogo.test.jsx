import React from 'react';
import TestRenderer from 'react-test-renderer';

import AirlineLogo from './AirlineLogo';

describe('AirlineLogo', () => {
    it('should render correctly', () => {
        const tree = TestRenderer.create(<AirlineLogo />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});