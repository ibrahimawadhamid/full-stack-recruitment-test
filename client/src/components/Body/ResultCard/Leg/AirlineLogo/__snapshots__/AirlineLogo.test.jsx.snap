// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AirlineLogo should render correctly 1`] = `
<div
  className="bpk-grid__column bpk-grid__column--2 bpk-grid__column--padded"
>
  <div
    className={null}
    style={
      Object {
        "width": 48,
      }
    }
  >
    <div
      className="bpk-image bpk-image--no-background"
      style={
        Object {
          "height": 0,
          "paddingBottom": "100%",
        }
      }
    >
      <img
        alt="airline logo"
        className="bpk-image__img"
        onLoad={[Function]}
        src="./EZ.png"
      />
    </div>
  </div>
</div>
`;
